Doc/library/dbm.rst

```diff
@@ -74,10 +75,10 @@ the Oracle Berkeley DB.
    modified by the prevailing umask).
 
 
-The object returned by :func:`.open` supports the same basic functionality as
-dictionaries; keys and their corresponding values can be stored, retrieved, and
-deleted, and the :keyword:`in` operator and the :meth:`keys` method are
-available, as well as :meth:`get` and :meth:`setdefault`.
+The object returned by :func:`open` supports the same basic functionality as a
+:class:`dict`; keys and their corresponding values can be stored, retrieved, and
+deleted, and the :keyword:`in` operator and the :meth:`!keys` method are
+available, as well as :meth:`!get` and :meth:`!setdefault`.
```

Comment:
```suggestion
deleted, and the :keyword:`!in` operator and the :meth:`!keys` method are
available, as well as :meth:`!get` and :meth:`!setdefault`.
```
Does `in` generate a link without the `!`?  Does adding the `!` work?

Reply:
Yes, it generates a link, and I think we want to keep it that way.